fbw-a32nx/src/systems/instruments/src/ISIS/PressureIndicator.tsx

```diff
@@ -28,8 +28,11 @@ export const PressureIndicator: React.FC = () => {
         }
     }, [baroUnitInHg, baroMode, hpaQnh, inHgQnh]);
 
+    const fontSize = baroMode === BaroMode.STD ? 36 : 28;
+    const positionX = baroMode === BaroMode.STD ? 250 : 256;
+
     return (
-        <text x={256} y={466} className="TextCyan" textAnchor="middle" fontSize={baroMode === BaroMode.STD ? 36 : 28}>
+        <text x={positionX} y={450} className="TextCyan" textAnchor="middle" fontSize={fontSize}>
```

Comment:
It's okay like this, but there's no problems with putting basic logic inline. You can achieve the same in the avionics framework with .map.

Reply:
sure, but in this case it would have pushed the line length quite a bit and in my view would have been less readable